package.json

```diff
@@ -224,7 +224,6 @@
     "precommit": "npm run -s install-if-no-packages && node bin/pre-commit-hook.js",
     "prepush": "npm run -s install-if-no-packages && node bin/pre-push-hook.js",
     "rm": "node bin/rm.js",
-    "postshrinkwrap": "node bin/trim-shrinkwrap.js && touch node_modules",
```

Comment:
`touch node_modules` was needed for `install-if-deps-outdated` to work. Please check if that's no longer needed.

Reply:
what's the expected behavior? the mtime on the node_modules directory should get updated after a `npm install` run?

Reply:
also, confirmed that `npm i --save some-package` updates npm-shinkwrap.json automatically.

Reply:
> the mtime on the node_modules directory should get updated after a npm install run?

Yes, but after doing something like `npm run update-deps` or `npm install --save left-pad`, both `node_modules` and `npm-shrinkwrap.json` are updated, roughly at the same time. For `install-if-deps-outdated` to work, `node_modules` mtime must be newer or equal.

Reply:
We are installing it already :)
https://github.com/Automattic/wp-calypso/pull/25374/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L253

Reply:
It may be better to run a one-liner like `node -e 'require("fs").utimesSync("./node_modules", new Date(), new Date())'` than installing the `cash-touch` dependency just for this. What do you think?

Reply:
+1 on the one-liner in node

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/25374/commits/43d45904f675c878d1aa27a105ed16125739715d